Utilities.BuildPipeline/Packages/com.utilities.buildpipeline/Editor/BuildInfo.cs

```diff
@@ -206,6 +206,10 @@ public virtual void ParseCommandLineArgs()
                     case "-acceptExternalModificationsToPlayer":
                         BuildOptions = BuildOptions.SetFlag(BuildOptions.AcceptExternalModificationsToPlayer);
                         break;
+                    case "-development":
+                        EditorUserBuildSettings.development = true;
```

Comment:
does this do something different from the next line?

Reply:
prob the same thing, but left it in to be safer than sorry.